cli/commands/bench.go

```diff
@@ -0,0 +1,37 @@
+package commands
+
+import (
+	"github.com/apex/log"
+	"github.com/spf13/cobra"
+	"github.com/tarantool/cartridge-cli/cli/bench"
+)
+
+func init() {
+	var benchCmd = &cobra.Command{
+		Use:   "bench",
+		Short: "Util for running benchmarks for Tarantool",
+		Long:  "Benchmark utility that simulates running commands done by N clients at the same time sending M simultaneous queries",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := bench.Run(ctx.Bench); err != nil {
+				log.Fatalf(err.Error())
+			}
+		},
+	}
+	rootCmd.AddCommand(benchCmd)
+
+	configureFlags(benchCmd)
+
+	benchCmd.Flags().StringVar(&ctx.Bench.URL, "url", "127.0.0.1:3301", "Tarantool address")
```

Comment:
Set dot at the end of the sentence (applies to the description of all parameters).

Reply:
Sorry, no. We keep patch copypasted from other carg codebase as possible. See https://github.com/tarantool/cartridge-cli/blob/master/cli/commands/admin.go

Reply:
Ok.